src/app/components/ThemeProvider/withThemeProvider.tsx

```diff
@@ -124,6 +124,11 @@ import {
 import gridWidths from './gridWidths';
 
 import { BrandPalette, Typography, BrandSVG } from '../../models/types/theming';
+import { PageTypes } from '../../models/types/global';
+
+const isDarkUiPage = (pageType: PageTypes, derivedPageType?: string | null) =>
+  pageType === 'mediaArticle' ||
+  (pageType === 'media' && derivedPageType?.toLowerCase() === 'on demand tv');
```

Comment:
Can we use the consts from https://github.com/bbc/simorgh/blob/latest/src/app/routes/utils/pageTypes.js here?



```suggestion
  pageType === MEDIA_ARTICLE ||
  (pageType === MEDIA && derivedPageType?.toLowerCase() === 'on demand tv');
```

Reply:
Although I do realise that there's a name clash between PageTypes (global type) and pageTypes - consts for each of the individual page types e.g. MAP, STY etc.